common/src/main/java/com/wynntils/features/overlays/ContentTrackerOverlayFeature.java

```diff
@@ -78,10 +78,12 @@ protected ContentTrackerOverlay() {
 
             toRenderPreview
                     .get(0)
-                    .setText(I18n.get("feature.wynntils.contentTrackerOverlay.overlay.tracker.title") + " Quest:");
+                    .setText(I18n.get("feature.wynntils.contentTrackerOverlay.overlay.contentTracker.title")
+                            + " Quest:");
```

Comment:
Technically Quest could be an i18n..

Reply:
I think we have hard coded texts for preview elsewhere.

Reply:
Oh previews.. nevermind then

Reply:
Like two lines below :-D.